### PR TITLE
[new][author] strapdown-topbar

### PR DESCRIPTION
**Pull request by joedf**
**Library name:** strapdown-topbar
**Git repository url:** https://github.com/joedf/strapdown-topbar
**License(s):** MIT License (see [source](https://github.com/joedf/strapdown-topbar/blob/gh-pages/assets/strapdown/strapdown-topbar.min.js#L2))
**Official homepage:** http://joedf.github.io/strapdown-topbar
**Wanna say something? Leave message here:** I am the author of this library and I would like to add it to cdnjs.

### CODE REVIEW

**Comment by nolsherry** [comment]
@joedf Please follow the instructions to [create the package.json](https://github.com/cdnjs/cdnjs/blob/master/README.md#create-or-update-packagejson), thanks!

**Comment by joedf** [comment]
Done. I've used the following as a template: https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/16pixels/package.json

**Comment by nolsherry** [comment]
@joedf please put the ```package.json``` file in to your project, and add the tags (v1.6.3) ,thanks!
image

**Comment by joedf** [comment]
Done :+1: 
https://github.com/joedf/strapdown-topbar/blob/v1.6.3/package.json

**Comment by nolsherry** [comment]
@joedf Please modify your commit messages to ```add strapdown-topbar@1.6.3 w/ git auto-update config ```, and use ```git commit --amend``` to remain only one commit record, thanks!

File: ajax/libs/strapdown-topbar/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "strapdown-topbar",
+  "filename": "strapdown-topbar.min.js",
+  "version": "1.6.3",
+  "description": "a topbar modification for strapdown.js",
+  "homepage": "http://joedf.github.io/strapdown-topbar",
```
Comment (pvnr0082t) [inline ajax/libs/strapdown-topbar/package.json]: @joedf I think it'll be better to use `https` for `github.io` domain. How do you think?

**Comment by joedf** [comment]
@pvnr0082t Changed it to `https` as you have suggested. However, it redirects to my *non-https* website.
@nolsherry I am currently cloning the `cndjs`repository so that I may finally run the git commands. All my changes so far were done online through the GitHub website, because I wanted to avoid this... :confounded: (_It's taking a very long time considering there are over 2.8 million items totalling over 25.9GB. And then, there's the file verification (checksums)... :sob:_ ).

**Comment by PeterDaveHello** [comment]
> However, it redirects to my non-https website.

If so, maybe we should use the redirected result?

**Comment by PeterDaveHello** [comment]
@joedf do the works online is fine, I'll squash for you.

**Comment by joedf** [comment]
@PeterDaveHello All of it is hosted on github. It's just that my custom domain using `CNAME` on my github.io website does not have an ssl certificate. My custom domain is subjected to change, however the current redirect is managed automatically by github. So no change is needed.

**Comment by PeterDaveHello** [comment]
@joedf Okay, that's finee.

**Comment by joedf** [comment]
@PeterDaveHello Many thanks!

**Comment by PeterDaveHello** [comment]
@joedf np

**Comment by PeterDaveHello** [comment]
@joedf is it possible to also add the tags for old versions ?

**Comment by joedf** [comment]
@PeterDaveHello Done! :+1: 
```
v1.0.0
v1.1.0
v1.2.0
v1.3.0
v1.4.1
v1.4.2
v1.4.3
v1.4.4
v1.5.0
v1.5.1
v1.5.2
v1.5.3
v1.5.4
v1.5.6
v1.6.0
v1.6.1
v1.6.3
```

**Comment by PeterDaveHello** [comment]
@joedf Thanks.

**Comment by joedf** [comment]
Great! :+1:

**Pull request merged**